crates/krun-server/src/bin/krun-server.rs

```diff
@@ -0,0 +1,31 @@
+use std::env;
+use std::process::Command;
+use std::thread;
+
+use anyhow::Result;
+use krun_server::cli_options::options;
+use krun_server::server::start_server;
+
+fn main() -> Result<()> {
+    env_logger::init();
+
+    let options = options().run();
+
+    let server_port = env::var("KRUN_SERVER_PORT").unwrap_or("3333".to_string());
```

Comment:
This could be in `cli_options` (`--port` / `-p`?)

`bpaf` has support for parsing env var too: https://docs.rs/bpaf/latest/bpaf/fn.env.html

Reply:
It's actually more convenient having it as an environment variable as it severs the double purpose of allowing us to know if krun is being run in the VM itself.

Reply:
Sure, but we don't have to do the parsing manually here if we could let `bpaf` do it for us? And it'd be more uniform when we add more options.

Reply:
Ah, yes, I didn't get that on the first read... 😅 Will update the PR.

Reply:
Done with bpaf, thanks!